src/demo/java/com/demo/amplitude/DemoController.java

```diff
@@ -1,13 +1,9 @@
 package com.demo.amplitude;
 
+import com.amplitude.*;
```

Comment:
Nit: Maybe we can keep the explicit imports. Will create a ticket to add a code style check for Java SDK later.

Reply:
Oh, this is in demo, I guess either is fine then.